src/QMCWaveFunctions/Fermion/DiracMatrixComputeCUDA.hpp

```diff
@@ -75,8 +74,8 @@ class DiracMatrixComputeCUDA : public Resource
   Vector<char, OffloadPinnedAllocator<char>> invM_ptrs_;
 
   // cuBLAS geam wants these.
-  DeviceValue<T_FP, OffloadPinnedAllocator<T_FP>> dev_one{1.0};
-  DeviceValue<T_FP, OffloadPinnedAllocator<T_FP>> dev_zero{0.0};
+  T_FP dev_one{1.0};
```

Comment:
Needs renaming. It is not device but host

Reply:
doh